docs/products/access/api-reference/config-api/service-providers-api.md

```diff
@@ -0,0 +1,333 @@
+# Service Providers API
+
+This allows the creation of new Service Provider via a REST API.
+It can be utilized in scripts to add, list, edit, or delete service providers.
+
+All endpoints are protected with API client. 
+It requires an API client with the `onegini_api_config` scope (Config API).
+
+## Endpoints
+
+### List of Service Providers
+
+This returns a list of all Service Providers.
+
+* Endpoint: `/api/v1/saml-sp-config`
+* Method: GET
+
+Example request:
+
+```http
+GET /api/v1/saml-sp-config HTTP/1.1
+Host: onewelcome.example.com
+```
+
+Example success response:
+
+```http
+HTTP/1.1 200 Ok
+Content-Type: application/json;charset=UTF-8
+Cache-Control: no-store
+Pragma: no-cache
+{
+   "result":[
+        {
+            "id": "e465d256-fd78-44db-8645-c23a78349251",
+            "name": "Service Provider name",
+            "entity_id": "https://service.provider.com",
+            "metadata_type": "XML"
+        },
+        {
+        ... more service providers ...
+        }
+   ]
+}
+```
+
+In the event of an error, one of the [generic error codes](#error-codes) will be returned.
+
+### Create Service Provider
+
+This creates a new Service Provider
+
+* Endpoint: `/api/v1/saml-sp-config`
+* Method: POST
+
+JSON body parameters:
+
+| Param                     | Required | Example                           | Description                                                                      |
```

Comment:
As this table is common for all methods, maybe it should be defined before `POST`

Reply:
done